trace-encoder/src/main/java/com/squareup/tracing/TraceEncoder.kt

```diff
@@ -109,23 +109,10 @@ public class TraceEncoder(
   ) {
     val timestampMicros = getTimestampNow()
     val chromeTraceEvents = event.toChromeTraceEvent(threadId, processId, timestampMicros)
-    events.safeOffer(listOf(chromeTraceEvents))
+    events.trySend(listOf(chromeTraceEvents))
   }
 
   private fun getTimestampNow(): Long = start.elapsedNow
-
-  /**
-   * Like [SendChannel.offer] but won't throw if the channel is closed.
-   */
-  private fun <T> SendChannel<T>.safeOffer(value: T) {
-    try {
-      trySend(value)
```

Comment:
This line was originally `offer`, but got updated to `trySend` last time coroutines were updated.  The `try/catch` no longer does anything since `trySend` doesn't throw.